bin/pynani.py

```diff
@@ -245,3 +233,20 @@ def server_start(port):
         print("To quit the server or to turn off exit the command-line")
         print("Thank you!")
         httpd.serve_forever()
+
+# Entry Point
+try:
+    if sys.argv[1] == "create-app":
+        try:
+            name = sys.argv[2]
+        except IndexError:
+            name = "My App"
+        
+        try:
+            description = sys.argv[3]
+        except IndexError:
+            description = "My PyNani application."
+        
+        create_app(name, description)
+except IndexError:
+    pynani_help()
```

Comment:
You can insert if on:

```
if __name__ == '__main__':...
```

Reply:
I also added the description as the 4th argument so you could create app now by using `pynani.py create-app MyApp "This is the description."`.